pom.xml

```diff
@@ -299,7 +299,7 @@ This Feature has been *DEPRECATED* and will be removed in a future version.
 are essentially embedding a slightly modified version of felix -->
         <osgi.version>5.0.0</osgi.version>
         <keyczar.version>0.66</keyczar.version>
-        <cas.client.version>3.4.1</cas.client.version>
+        <cas.client.version>3.2.2</cas.client.version>
```

Comment:
We need a cas bug fix in this version.

Reply:
Do you have a link to a CAS issue and a DDF issue to reference for the dependency upgrade?

Reply:
There's no issue yet, since we're not sure if this is really a bug, but the relevant CAS commmit is https://github.com/apereo/java-cas-client/commit/be2a64147320aacc6262b5fd181bea46e02618cd

Reply:
There's a comment on that commit from somebody else who ran into the same issue.